lib/bolt/outputter/human.rb

```diff
@@ -174,11 +174,22 @@ def print_result(result)
           if result.message?
             @stream.puts(remove_trail(indent(2, result.message)))
           end
-
           case result.action
           when 'command', 'script'
             safe_value = result.safe_value
-            @stream.puts(indent(2, safe_value['merged_output'])) unless safe_value['merged_output'].strip.empty?
+            if safe_value["merged_output"]
+              @stream.puts(indent(2, safe_value['merged_output'])) unless safe_value['merged_output'].strip.empty?
+
+            else # output stdout or stderr
+              unless safe_value['stdout'].nil? || safe_value['stdout']&.strip&.empty?
```

Comment:
```suggestion
              unless safe_value['stdout'].nil? || safe_value['stdout'].strip.empty?
```

If this is a non-nil value, it'll be a string. No need for safe operators then.

Reply:
I read them backwards, you can ignore my question. Thanks for clarification!